inst/include/plant/models/ff16_environment.h

```diff
@@ -42,15 +41,12 @@ class FF16_Environment : public Environment {
   }
 
   double get_environment_at_height(double height) const {
```

Comment:
So this just calculates light availability at this point? Will the method be expanded to return all environmental values at a given height (i.e. + soil moisture) or will this always be with respect to light. There will eventually be a completely new strategy so no need for this function to refer to anything but light availability I suppose.

Reply:
Yes, may eventually return a vector. In FF16 it's just a single number, light availability.